src/support.js

```diff
@@ -337,6 +344,24 @@ function relocateExports(exports, memoryBase, tableBase, moduleLocal) {
   return relocated;
 }
 
+#if RELOCATABLE
+// Dynmamic version of shared.py:make_invoke.  This is needed for invokes
+// that originate from side modules since these are not known at JS
+// generation time.
+function createInvokeFunction(sig) {
+  return function() {
+    var sp = stackSave();
+    try {
+      return Module['dynCall_' + sig].apply(null, arguments);
```

Comment:
How do we know that the appropriate dynCalls will exist?

Reply:
Because dynCalls wasm-emscripten-finalize ensures that for every address taken function that dynCall with a corresponding signature is created.   In many cases the dynCall will exist in both the side module and main module, but its should never be the case the it doesn't exist at all.

At least I've not found a test case where it doesn't.   

The bug in question exists because the invoke were missing, not the dynCalls they correspond to.

Reply:
Oh ok, so the issue is a dynCall that exists in the side module, but not in the main module? Makes sense. Maybe worth a comment.